test/keystore.spec.ts

```diff
@@ -25,6 +28,15 @@ describe(`Keystore`, () => {
     ).toBeDefined()
   })
 
+  it(`check network is set`, () => {
+    storeWallet(mockWallet4, 'mock-name4', 'mock-password', 'regen-testnet')
+    expect(
+      localStorage.getItem(`cosmos-wallets-xrn:1h0y77r8ee28hs0wqg9css7rzegmagaamwl6rdp`)
+    ).toBeDefined()
+    const wallet = JSON.parse(localStorage.getItem(`cosmos-wallets-xrn:1h0y77r8ee28hs0wqg9css7rzegmagaamwl6rdp`) || '[]')
```

Comment:
why are you defaulting to an array here?

```suggestion
    const wallet = JSON.parse(localStorage.getItem(`cosmos-wallets-xrn:1h0y77r8ee28hs0wqg9css7rzegmagaamwl6rdp`) || '{}')
```